tests/Http/PostTest.php

```diff
@@ -1102,6 +1102,10 @@ public function testCreatingVoterRegistrationPost()
         // Mock the Blink API call.
         $this->mock(Blink::class)->shouldReceive('userSignupPost');
 
+        // Mock the Northstar API calls.
+            $this->mock(Northstar::class)
```

Comment:
It isn't necessary to re-mock Northstar here since we handle all calls to `getUser` in the base mock.